css/style.css

```diff
@@ -0,0 +1,722 @@
+  font-style: normal;
+  font-weight: 200;
+  font-size: 18px;
+  line-height: 30px;
+  margin-bottom: 45px;
+}
+.deco{
+  background: var(--primary);
+  padding: 10px 20px;
+  width: 25%;
+  margin-bottom: 15px;
+}
+.experience{
+  margin-bottom: 40px;
+}
+.experience-section{
+  margin-top: 30px;
+}
+.experience-section h2{
+  font-family: var(--font-playfair-display),var(--font-serif);
+  font-style: normal;
+  font-weight: 900;
+  font-size: 32px;
+  line-height: 42px;
+}
+.experience-role{
+  font-family: var(--font-narrow),var(--font-serif);
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  color: var(--gold);
+  margin-bottom: 15px;
+}
+.experience-date{
+  font-family: var(--font-narrow),var(--font-serif);
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 24px;
+  color: var(--black);
+  margin-bottom: 10px;
+}
+.education{
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+.education h1{
+  font-family: var(--font-playfair-display),var(--font-serif);
+  font-style: normal;
+  font-weight: 900;
+  font-size: 36px;
+  line-height: 46px;
+  color: var(--black);
+  margin-bottom: 20px;
+}
+.school{
+  font-family: var(--font-narrow),var(--font-serif);
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  color: var(--gold);
+  margin-bottom: 10px;
+}
+.grad-date{
+  font-family: var(--font-narrow),var(--font-serif);
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 24px;
+  color: var(--black);
+  margin-bottom: 6px;
+}
+.skill-set{
+  padding: 30px;
+  padding-top: 8px;
+}
+.skill-set-section h3{
+  font-family: var(--font-narrow),var(--font-serif);
+  font-style: normal;
+  font-weight: bold;
+  font-size: 25px;
+  line-height: 24px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: var(--gold);
+  margin-bottom: 8px;
+}
+.skill-set-list{
+  padding: 15px 0px;
+  margin-bottom: 15px;
+}
+.skill-set-list-item{
+  padding: 3px 0px;
+  font-family: var(--font-narrow),var(--font-serif);
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 30px;
+  color: var(--black);
+  
+}
+.awards-section h3{
+  font-family: var(--font-narrow), var(--font-serif);
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: var(--gold);
+}
+.awards-list {
+  padding: 12px 0px;
+  margin-bottom: 12px;
+}
+.awards-list-item{
+  padding: 3px 0px;
+  font-family: var(--font-narrow),var(--font-serif);
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 30px;
+  color: var(--black);
+}
+.career-traits{
+  margin-bottom: 35px;
+}
+.career-traits h3{
+  font-family: var(--font-narrow),var(--font-serif);
```

Comment:
Looks like you've written this so many times. Maybe there's a better way to group all the styles for font family into one? Something like 

```css
h3, .awards-list-item ..... { font-family: var(...) }
```